src/components/auth/GoogleButton/googleButton.module.scss

```
@@ -0,0 +1,41 @@
+@import '@/styles/variables';
+@import '@/styles/fonts';
+@import '@/styles/mixins';
+
+.googleButton {
+  width: 100%;
+  background-color: $text-color;
+  border-radius: $p-1-5;
+  cursor: pointer;
+
+  padding: $p-3;
+
+  display: flex;
+  justify-content: center;
+  gap: $p-4;
+
+  :hover {
+    opacity: 0.6;
+  }
```

suggestion: to have a smooth animation, we could use transition from css
```suggestion
  gap: $p-4;
	transition: 2s ease-out;

  :hover {
    opacity: 0.6;
  }
```